src/solver/variable/economy/max-mrg.cpp

```diff
@@ -1,187 +0,0 @@
-/*
```

Comment:
This file was removed.
It contains code for detailed computation of **max margin** output variable.  
This code was moved into a couple of source files **max-mrg-utils.h/.cpp**.
Note that this code was changed in order to : 
- isolate the part of computation that is common to **classic** and **CRS** max margin computation  
-  introduction std algorithm where needed

See comments in files **max-mrg-utils.h/.cpp** for more details.